guides/doc-Upgrading_and_Updating/topics/migrating_pulp_content.adoc

```diff
@@ -0,0 +1,71 @@
+[[id="migrating_pulp_content"]]
+
+= Migrating Content to Pulp 3
+
+The time to prepare content for Pulp 3 depends on the amount of content and the number of content views.
+For large systems, this can mean several days of downtime.
+To prevent this, {Team} recommends that you pre-migrate Pulp content while running the latest version of {ProjectServer} {ProjectVersionPrevious}.
```

Comment:
I was told `recommend` is a problematic wording.

Maybe as an alternative say something like "pre-migrate content to reduce downtime/avoid overly long update procedures"?

cc @melcorr

Reply:
You're correct @maximiliankolb 
IBM Style, which is the standard with which all existing docs were written and reviewed against, cautions against "recommend" on pg 357:

"recommend v
Avoid if possible. A phrase such as “We recommend that you take the following action” could create a
potential marketing or legal problem."
Thank you for contributions!

Reply:
revmoved